.github/actions/spelling/allow.txt

```diff
@@ -37,3 +37,10 @@ uilt
 Windo
 ELSPROBLEMS
 requ
+whatif
+pscustomobject
+itsdangerouss
+itsdangerouss
+Bandwith
+PCs
+wmiprvse
```

Comment:
This one should probably be in `expect/software.txt`

You don't have to change it, it's just a preference to try and keep things organized by what type of word they are. I'll have to update the advice file to give better guidance, but the way it's structured is that `expect` is for "real" words / terms that appear semi-frequently, and `allow` is for the weird one-offs like `itsdangerouss`